### PR TITLE
nimble/bttester: Make adv parameters global & small fixes

### PR DESCRIPTION
**Pull request by szymon-czapracki**
MYNEWT_VAL is corrected in multiple places for an up-to-date name. 
Missing semicolon and invalid function name are fixed in btp_bap.c
Change codec configuration to be set on BIS level.
Add necessary syscfg options for BAP tests.

### CODE REVIEW

**Comment by szymon-czapracki** [comment]
@KKopyscinski @sjanc FYI

**Review by KKopyscinski — APPROVED** [review]

File: apps/bttester/syscfg.yml [hunk 1]
```diff
@@ -97,6 +97,8 @@ syscfg.vals:
     BLE_ISO_MAX_BISES: 1
     BLE_ISO_MAX_BIGS: 1
     BLE_EXT_ADV: 1
+    BLE_PHY_2M: 1
+    BLE_EXT_ADV_MAX_SIZE: 261
```
Comment (sjanc) [inline apps/bttester/syscfg.yml:101]: why is this needed? it is quite big value

Comment (szymon-czapracki) [reply apps/bttester/syscfg.yml]: Updated the value to `40` - previous value was an overkill. This is necessary for broadcast, and specially extended advertising configuration in broadcast bttester application.

**Review by sjanc — APPROVED** [review]
